### PR TITLE
refactor: add type options for attribute StudioTemplateProperty

### PR DESCRIPTION
**Pull request by HegelPro**
affects: @haulmont/jmix-front-generator

### CODE REVIEW

File: packages/jmix-front-generator/src/common/studio/studio-model.ts [hunk 1]
```diff
@@ -3,37 +3,111 @@ import {RestService, RestServiceMethod} from "../model/cuba-model";
...
-  REST_SERVICE_METHOD = 'REST_SERVICE_METHOD',
   POLYMER_COMPONENT_NAME = 'POLYMER_COMPONENT_NAME',
   PASSWORD = 'PASSWORD',
   GRAPHQL_QUERY = 'GRAPHQL_QUERY',
   MENU_ITEM = 'MENU_ITEM',
   ATTRIBUTE = 'ATTRIBUTE',
   ATTRIBUTES_ARRAY = 'ATTRIBUTES_ARRAY',
+
+  // deprecated types
+  MULTI_OPTION = 'MULTI_OPTION',
```
Comment (vyacheslav-pushkin) [inline packages/jmix-front-generator/src/common/studio/studio-model.ts:18]: Why is multi-option deprecated?

File: packages/jmix-front-generator/src/common/studio/studio-model.ts [hunk 2]
```diff
@@ -3,37 +3,111 @@ import {RestService, RestServiceMethod} from "../model/cuba-model";
...
+}
+
+export interface IntegerStudioTemplateProperty extends BaseStudioTemplateProperty {
+  propertyType: StudioTemplatePropertyType.INTEGER;
+}
+
+export interface OptionStudioTemplateProperty extends BaseStudioTemplateProperty {
+  propertyType: StudioTemplatePropertyType.OPTION;
+  options: string[];
+}
```
Comment (vyacheslav-pushkin) [inline packages/jmix-front-generator/src/common/studio/studio-model.ts:56]: I'd suggest to revert this change for now, I'm not 100% sure it won't break anything in Studio.

**Review by vyacheslav-pushkin — CHANGES_REQUESTED** [review]

File: packages/jmix-front-generator/src/common/studio/studio-model.ts (hunk 1 above)
Comment (HegelPro) [reply packages/jmix-front-generator/src/common/studio/studio-model.ts:18]: In our templates, we don't use it. I can keep it as common studio property.

File: packages/jmix-front-generator/src/common/studio/studio-model.ts (hunk 2 above)
Comment (HegelPro) [reply packages/jmix-front-generator/src/common/studio/studio-model.ts:56]: I didn't change existing logic, just define typescript type for every studio property inside the generator. I thought that Studio don't see typescript types.

File: packages/jmix-front-generator/src/common/studio/studio-model.ts (hunk 1 above)
Comment (vyacheslav-pushkin) [reply packages/jmix-front-generator/src/common/studio/studio-model.ts:18]: I think it shouldn't be deprecated.

**Review by vyacheslav-pushkin — APPROVED** [review]
